src/filesNodes.ts

```diff
@@ -0,0 +1,53 @@
+;/*---------------------------------------------------------------------------------------------

```

Comment:
what's with the semicolon? It shows up at the top of a few files where you added the copyright notice...